Tests/OpenChat.API.AcceptanceTests/RegistrationTests.cs

```diff
@@ -0,0 +1,42 @@
+using System.Net.Http;
+using Xunit;
+using Xunit.Abstractions;
+using AspNetCore.Http.Extensions;
+using System.Threading.Tasks;
+using FluentAssertions;
+using System.Net;
+using Newtonsoft.Json.Linq;
+using System;
+
+namespace OpenChat.API.AcceptanceTests
+{
+    public class RegistrationTests : ApiTests
```

Comment:
Does it make more sense to rename this class to "RegistrationApiTests"? The acceptance tests are testing API from the outside.

Reply:
yes

Reply:
why not renaming this class UsersApiTests? since we're testing the UsersController

Reply:
or just UsersControllerTests, because we're in a project dedicated to API tests already (we don't need to be redundant in class naming)

Reply:
I might be biased towards the way apis are grouped in the swagger file - by the tags property. The tags from the swagger file are: registration, user, login, post.

Reply:
OK I see. Grouping them differently than how the controllers are being grouped means we're defining a different domain in the API Tests layer, meanwhile the back end would have a different domain - maybe we should keep these two consistent?

Reply:
it's the easiest way to tell "this set of tests is linked to this API that has this responsibility"

Reply:
it's the easiest way to tell "this set of tests is linked to this API that has this responsibility"

Reply:
> it's the easiest way to tell "this set of tests is linked to this API that has this responsibility"

Should this API name be tightly coupled with the controller handling the request? Changing the name of the controller for some reason will cause the test class name to change.

Reply:
technically nobody will require you to change the test name as well (on API tests, but even on unit tests), build will still work;
but if we want to keep that convention (because of above mentioned reasons), then yeah we'll have to rename tests manually as well if we want to group them same way as in the back end